notebooks/search/07-inference.ipynb

```diff
@@ -0,0 +1,380 @@
+    "the OpenAI service. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "15193c10",
+   "metadata": {},
+   "source": [
+    "# Create Elastic Cloud deployment\n",
+    "\n",
+    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook) for a free trial.\n",
+    "\n",
+    "- Go to the [Create deployment](https://cloud.elastic.co/deployments/create) page\n",
+    "   - Under **Advanced settings**, go to **Machine Learning instances**\n",
+    "   - You'll need at least **4GB** RAM per zone for this tutorial\n",
+    "   - Select **Create deployment**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f27dffbf",
+   "metadata": {},
+   "source": [
+    "# Install packages and connect with Elasticsearch Client\n",
+    "\n",
+    "To get started, we'll need to connect to our Elastic deployment using the Python client.\n",
+    "Because we're using an Elastic Cloud deployment, we'll use the **Cloud ID** to identify our deployment.\n",
+    "\n",
+    "First we need to `pip` install the following packages:\n",
+    "\n",
+    "- `elasticsearch`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c4b16bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install elasticsearch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "41ef96b3",
+   "metadata": {},
+   "source": [
+    "Next, we need to import the modules we need. 🔐 NOTE: getpass enables us to securely prompt the user for credentials without echoing them to the terminal, or storing it in memory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "690ff9af",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from elasticsearch import Elasticsearch, helpers\n",
+    "from urllib.request import urlopen\n",
+    "import getpass\n",
+    "import json\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "23fa2b6c",
+   "metadata": {},
+   "source": [
+    "Now we can instantiate the Python Elasticsearch client.\n",
+    "\n",
+    "First we prompt the user for their password and Cloud ID.\n",
+    "Then we create a `client` object that instantiates an instance of the `Elasticsearch` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "195cc597",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Found in the 'Manage Deployment' page\n",
+    "CLOUD_ID = getpass.getpass('Enter Elastic Cloud ID:  ')\n",
+    "\n",
+    "# Password for the 'elastic' user generated by Elasticsearch\n",
+    "ELASTIC_PASSWORD = getpass.getpass('Enter Elastic password:  ')\n",
+    "\n",
+    "# Create the client instance\n",
+    "client = Elasticsearch(\n",
+    "    cloud_id=CLOUD_ID,\n",
+    "    basic_auth=(\"elastic\", ELASTIC_PASSWORD)\n",
+    ")"
```

Comment:
Take a look at any of the other notebooks in this directory to see how we are authenticating the client using an API key. I suggest we change this to follow the same pattern for consistency. Also if you could make sure that the prompt messages in the `getpass()` calls match exactly to those in the other notebooks that would be awesome as that will help us eventually test the notebook automatically.